imagersite/imager_images/models.py

```diff
@@ -1,11 +1,15 @@
+"""Make the models for the Photos and Albums."""
+
 from django.db import models
 from django.contrib.auth.models import User
 from sorl.thumbnail import ImageField
 
 
 # Create your models here.
+
 class Photo(models.Model):
-    """this the the Photo model"""
+    """This is the Photo model."""
+
     user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='photos')
     albums = models.ManyToManyField('Album', related_name='photos')
```

Comment:
This field should probably be on the `Album` model instead. Currently, in order to create a `Photo` you need to first create an `Album` to add it to.